src/python/foglamp/data_purge/sqlalchemy_purge.py

```diff
@@ -69,6 +60,24 @@
                                                                           time.localtime(time.time()))))
 
 
+"""logging table is instead of the log. After much thought, in addition to the discussed information the table also 
+includes the following: 
+    -> table to specify which table has been purged, since the process could occur in multiple tables 
+    -> total_unsent_rows specifies the total number of unsent rows that existed within range prior to the purge. 
+    based off that, and unsent_rows_removed one can calculate how many (unsent rows) remain.
+"""
+_LOGGING_TABLE = sqlalchemy.Table('purge_logging', sqlalchemy.MetaData(),
```

Comment:
This needs to go into the table defined in the schema and the architecture document